StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethod.swift

```diff
@@ -117,6 +117,13 @@ public class STPPaymentMethod: NSObject, STPAPIResponseDecodable {
     )
     @objc private(set) public var metadata: [String: String]?
 
+    /// The payment details of a PaymentMethod that was created using Link.
+    @_spi(STP) public var linkPaymentDetails: LinkPaymentDetails?
```

Comment:
This could also go into `STPPaymentMethodLink`. Since it's not part of the API data model, having it separate felt a little better, but happy to change if anyone feels strongly about it.